### DISCUSSION TITLE
Best way to exclude a group of posts from home page

### DISCUSSION THREAD

**Discussion by Iduoad** [adityatelange/hugo-PaperMod#1188] [Q&A] [answered]
I have a `content/locations/` with a bunch of posts and I need to exclude them from displaying in the home page. What is the best way to do that ?

**Comment by Kuglikrug** [comment, marked as answer]
Hi, add this to the pages you want to hide from the home page:

```
---
hiddenInHomeList: true
---
```

also check this out: https://github.com/adityatelange/hugo-PaperMod/discussions/724

**Reply by Iduoad** [reply]
It worked! Thank you!
